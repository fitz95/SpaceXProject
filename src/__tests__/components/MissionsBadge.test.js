import React from 'react';
import renderer from 'react-test-renderer';

import MissionsBadge from 'src/components/MissionsBadge';

describe('MissionsBadge', () => {
  it('renders correctly for reserved state', () => {
    const tree = renderer.create(<MissionsBadge isReserved />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('renders correctly for not reserved state', () => {
    const tree = renderer.create(<MissionsBadge isReserved={false} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
