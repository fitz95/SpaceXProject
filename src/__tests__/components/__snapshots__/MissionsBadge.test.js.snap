// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MissionsBadge renders correctly for not reserved state 1`] = `
<div
  className="label"
  style={
    Object {
      "backgroundColor": "#6d757d",
    }
  }
>
  NOT A MEMBER
</div>
`;

exports[`MissionsBadge renders correctly for reserved state 1`] = `
<div
  className="label"
  style={
    Object {
      "backgroundColor": "#18a2b8",
    }
  }
>
  Active Member
</div>
`;
